moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua

```diff
@@ -46,9 +46,9 @@ events = {
         description = "IAM action in production account from console without mfa",
         fields = {
             { "eventSource", "^iam.amazonaws.com$" },
-            { "recipientAccountId", "^1234567890$" },
+            { "recipientAccountId", "^1234567890$", "^1122954321$" },
             { "userIdentity.invokedBy", "^signin.amazonaws.com$" },
-            { "userIdentity.sessionContext.attributes.mfaAuthenticated", "^false$" }
+            { "userIdentity.sessionContext.attributes.mfaAuthenticated", "[^true]" }
```

Comment:
The negated set is a misleading/bad configuration example and is a hack that wouldn't work if you wanted to negate something like recipientAccountId.

Reply:
That is really good to know, thanks. Will remove for this PR and look into switching to lpeg.re.